boa_engine/src/value/conversions.rs

```diff
@@ -54,6 +54,18 @@ impl From<f64> for JsValue {
     }
 }
 
+impl From<f32> for JsValue {
+    #[allow(clippy::float_cmp)]
+    #[inline]
+    fn from(value: f32) -> Self {
+        // if value as i32 as f32 == value {
+        //     Self::Integer(value as i32)
+        // } else {
+        Self::Rational(value.into())
+        // }
```

Comment:
Why isn't this mirroring the `f64` conversion?

Reply:
It is the same as the `f64` one no?

Reply:
Ah, right, forgot we removed the `f64` conversion to integer